tests/suites/test_suite_psa_crypto.data

```diff
@@ -704,6 +768,78 @@ Copy fail: AES, incompatible target policy
 depends_on:MBEDTLS_AES_C:MBEDTLS_CIPHER_MODE_CTR:MBEDTLS_CIPHER_MODE_CBC
 copy_fail:PSA_KEY_USAGE_COPY | PSA_KEY_USAGE_EXPORT:PSA_ALG_CTR:0:PSA_KEY_TYPE_AES:"404142434445464748494a4b4c4d4e4f":0:0:PSA_KEY_USAGE_EXPORT:PSA_ALG_CBC_NO_PADDING:0:PSA_ERROR_INVALID_ARGUMENT
 
+Copy key: source=MAC min-length, target=MAC length > min-length
+depends_on:MBEDTLS_SHA256_C
```

Comment:
Running `psa_set_test_dependencies.py` didn't make any changes...

Reply:
You need to run `tests/scripts/set_psa_test_dependencies.py tests/suites/test_suite_psa_crypto.data`. This isn't automatic yet because `set_psa_test_dependencies.py` is somewhat experimental and we might not keep it in its present form: it was primarily intended as a one-off to rename the existing manual dependencies.

Reply:
Done